src/tests/getUserRequests.test.js

```diff
@@ -109,18 +109,6 @@ describe('/trip/myrequests', () => {
         done();
       });
   });
```

Comment:
@VedaDom any reason why this test case is omitted?

Reply:
I forgot remove this test, but the reason why it commented is that there was no response with the expected results.